queries.sql

```diff
@@ -8,3 +9,45 @@ SELECT NAME, ESCAPE_ATTEMPTS FROM animals WHERE WEIGHT_KG>10.5;
 SELECT * FROM animals WHERE NEUTERED=true;
 SELECT * FROM animals WHERE NAME!='Gabumon';
 SELECT * FROM animals WHERE WEIGHT_KG BETWEEN 10.4 AND 17.3;
+
+/* Task Two */
+
+BEGIN;
+
+UPDATE animals SET SPECIES='unspecified';
+
+ROLLBACK;
+
+BEGIN;
+
+UPDATE animals SET SPECIES='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET SPECIES='pokemon' WHERE species IS NULL;
+
+COMMIT;
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
```

Comment:
- [ ] Similarly here, please provide a query that **_verifies_** that the change was effected before and after the `rollback`.